Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM        sdurrheimer/alpine-golang-make-onbuild
+FROM        prom/busybox:glibc
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
-EXPOSE      9100
+COPY node_exporter  /bin/node_exporter
+
+EXPOSE     9100
+ENTRYPOINT [ "/bin/node_exporter" ]
```

Comment:
`sdurrheimer/alpine-golang-make-onbuild` was also using a entrypoint (`ENTRYPOINT  ["/bin/go-run"]`) :)

I like to use entrypoints when a image has a well defined objective, here to run our project binary which makes the image acts as a binary. @matthiasr any opinion on that ?